docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 1
+sidebar_label: "Configuration"
+id: daemon-config
+title: Configuration
+description: How to configure the Kwil Daemon
+slug: /daemon/config/
+---
+
+This section outlines the complete set of configurations available for the operation of a kwild node. The configurations are categorized into three distinct types: chain-specific configurations that pertain to the underlying blockchain functionality, app-specific configurations that govern the behavior of kwild application and logging. 
+
+Lets take a look into the different sections of the config. 
+
+## logging
+
+- `level`: This variable sets the logging level of the daemon. The default level is info.
+
+- `output_paths`: This variable sets the output paths for logs. Multiple paths can be specified, separated by commas. By default, logs are written to stdout.
+
+- `format`: This variable sets the logging format. Supports 'plain' or 'json'.
+
+- `time_format`: This variables sets the time format for logs. Available options are **epochfloat** (default), **epochmilli**, or **rfc3339milli**.
+
+## app   
+This section contains configuration specific to the operation of kwild application.
+
+- `grpc_listen_addr`: This variable sets the address at which the gRPC server of the daemon listens for connections. The default value is :50051.
+
+- `http_listen_addr`: This variable sets the address at which the HTTP server of the daemon listens for connections. The default value is :8080.
+
+- `sqlite_file_path`: This variable sets the file path for all SQLite files. If not provided, it defaults to `data/kwil.db/` under the specified root_dir.
+
+- `extension_endpoints`: This variable defines endpoints for extensions. Multiple endpoints can be specified, separated by commas.
+
+- `tls_cert_file`: This variable sets the path to a file containing certificate that is used to create the HTTPS RPC server. The path can be either an absolute path or path relative to the configured home_dir. If not provided, only http server would be created. 
+
+- `tls_key_file`: This variable sets the path to a file containing matching private key that is used to create the HTTPS RPC server. The path can be either an absolute path or path relative to the configured home_dir. If not provided, only http server would be created.
+
+- `hostname`: This variable sets the kwild server hostname.
+
+## chain:
+This section contains configuration specific to cometbft based blockchain.
+
+- `moniker`: This variable sets a custom human-readable name for the node.
+
+### rpc
+This subsection contains the configuration required to manage blockchain's rpc service.
+
+- `laddr`: This variable sets the TCP or UNIX socket address for the RPC server to listen on. Default listening address is on "tcp://127.0.0.1:26657"
+
+- `timeout_broadcast_tx_commit`: This variable sets the time to wait for a transaction to be committed during */broadcast_tx_commit*. Default timeout is 10s.
```

Comment:
Seems like a potential typo with the "/" in `/broadcast_tx_commit`. I'm also still not sure what this does based on this description (I don't know what `/broadcast_tx_commit` is.  Could you explain this?).